tests/p2p/config.yaml

```diff
@@ -28,11 +28,12 @@ bootstrap:
     layer-duration-sec: '60'
     layer-average-size: '20'
     hare-wakeup-delta: '10'
+    executable-path: '/bin/go-p2p'
   deployment_type: deployment # default:deployment
 # IMPORTANT: all clients on the same suite should contain the same args (hare-committee-size, hare-max-adversaries etc)!
 client:
-  image: 'spacemeshos/go-spacemesh:develop'
-  command: '/bin/go-p2p'
+  image: 'spacemeshos/go-spacemesh-dev:harness'
```

Comment:
dont push these changes. image should stay develop

Reply:
fixed in 704b0c0b747b960ba1d5920e85d81182eaa660a8